Styles.css

```diff
@@ -282,6 +292,87 @@
   background-color: var(--clothing-red-color);
 }
 
+.testimonials__carousel {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-top: calc(var(--margin-spacer) * 3);
+  font-size: calc(var(--font-size) * 1);
+  display: none;
+  transition-duration: 2s;
+}
+
+.testimonials__header {
+  margin-top: calc(var(--margin-spacer) * 3);
+  text-align: center;
+}
+
+.testimonials__subheader {
+  font-weight: 300;
+  margin-top: calc(var(--margin-spacer) * 1);
+  margin-bottom: calc(var(--margin-spacer) * 2);
+  font-size: calc(var(--font-size) * 1);
+  text-align: center;
+}
+
+.testimonials__carousel__picture {
+  height: 80px;
+  width: 80px;
+  margin-bottom: calc(var(--margin-spacer) * 2);
+  border-radius: 50%;
+}
+
+.testimonials__carousel__description {
+  display: flex;
+  justify-content: center;
+  max-width: 30em;
+  text-align: center;
+  font-weight: 300;
+  font-size: calc(var(--font-size) * 1.25);
+  margin-left: calc(var(--margin-spacer) * 3);
```

Comment:
this can be written in a better way without having to write both left and right
you can do margin: 0 calc(var(--margin-spacer) * 3);